code/src/component/MovieList.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react'
+
+import { MovieContainer } from 'component/MovieContainer';
+import { Nav } from 'component/Nav'
+
+export const MovieList = ({ list }) => {
+    const [movies, setMovies] = useState([]);
+
+    const API_KEY = '7a8b8f73f0f19bc86674c69db87cbbc7'
+    const MOVIE_URL = `https://api.themoviedb.org/3/movie/${list}?api_key=${API_KEY}&language=en-US&page=1`;
+  
```

Comment:
So clever to pass in the list prop and put that in the MOVIE_URL to fetch the different lists, great job! 🙌